clients/python/llmengine/model.py

```diff
@@ -12,27 +12,29 @@ class Model(APIEngine):
     """
     Model API. This API is used to get, list, and delete models. Models can be created using [FineTune.create()](./#llmengine.fine_tuning.FineTune.create).
 
+    Our DTOs mention "endpoint"s, which are our concept of a physical running instance of a Model. There is a 1:1 correspondence between Models and endpoints, so you can think of them as equivalent.
```

Comment:
Maybe say "response objects" instead of "DTOs"? Granted the user is going to be technical, but probably want to be a bit more formal in this setting.